README.md

```diff
@@ -2,3 +2,58 @@ Scalanus
 ========
 
 Yet another scripting language. Basically [Janus](https://github.com/mkaput/janus) done better, faster and funnier. Now in Scala. Much scalable. Very advanced. Such wow. Project for Scala Course at AGH UST.
+
+## TODO
+
+| | |
+|---|---|
+| 💔 | not implemented |
+| ⚙️ | in progress |
```

Comment:
To samo co wcześniej. To jest bardziej lista dla odwiedzających repo niż dla nas